packages/web/src/common/store/cache/tracks/sagas.ts

```diff
@@ -291,11 +291,14 @@ function* confirmEditTrack(
         )
         yield* call(recordEditTrackAnalytics, currentTrack, confirmedTrack)
       },
-      function* () {
+      function* ({ error, message, timeout }) {
+        const reportToSentry = yield* getContext('reportToSentry')
+        reportToSentry({
+          error,
+          additionalInfo: { message, timeout, trackId, formFields },
+          name: 'Edit Track Failed'
```

Comment:
Nitpicking but I think `EditTrack` probably fits better with current other usages (we know it's failed because it's an error), and would be easier to filter/search by

Reply:
Works for me! I think I was trying to keep the error name similar to what already existed. But I now realize that in Sentry, the existing errors were just named `Error`, so this doesn't really matter.